src/Resources/app/administration/src/config/hipay-check-server-access/index.js

```diff
@@ -58,7 +58,7 @@ Component.register('hipay-check-server-access', {
     },
     getConfig() {
       let systemConfigComponent = this.$parent;
-      while (!systemConfigComponent.hasOwnProperty('actualConfigData')) {
+      while (systemConfigComponent.actualConfigData === undefined) {
```

Comment:
Prefer to use typeof _my_var_ === 'undefined'

Reply:
Although the proposed solution is not the most optimal, it is the solution that works with the minimum effort considering the update of the vue version from 2 to 3. I also tested the solution you proposed but it does not work. Most likely, in the future, a larger refactoring will be necessary.

Reply:
No problem, if the actual solution works with Vue 3, I accept